netmon.cc

```diff
@@ -57,12 +57,13 @@ CheckResult TCPPortClosedChecker::perform()
 
 HTTPSChecker::HTTPSChecker(sol::table data) : Checker(data)
 {
-  checkLuaTable(data, {"url"}, {"maxAgeMinutes", "minBytes", "minCertDays", "serverIP"});
+  checkLuaTable(data, {"url"}, {"maxAgeMinutes", "minBytes", "minCertDays", "serverIP", "method"});
   d_url = data.get<string>("url");
   d_maxAgeMinutes =data.get_or("maxAgeMinutes", 0);
   d_minCertDays =  data.get_or("minCertDays", 14);
   string serverip= data.get_or("serverIP", string(""));
   d_minBytes =     data.get_or("minBytes", 0);
+  string d_method= data.get_or("method", string("GET"));
```

Comment:
this needs to check if it is GET or HEAD because we don't support anything else.